lang/cs/Org.Apache.REEF.IMRU/API/UpdateResult.cs

```diff
@@ -50,22 +50,22 @@ internal bool IsNotDone
             get { return !_done; }
         }
 
-        internal bool HasMapInput
+        public bool HasMapInput
```

Comment:
UpdateResult is already accessible form IMRU Example. User task function uses it to hold the result. The task needs to access those property so that to know what kind of result it contains at the moment in task state management. 
I will add the document.